[2023-12-16T14:22:29.145+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_data_pipeline.extract_data manual__2023-12-16T14:22:27.349459+00:00 [queued]>
[2023-12-16T14:22:29.154+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_data_pipeline.extract_data manual__2023-12-16T14:22:27.349459+00:00 [queued]>
[2023-12-16T14:22:29.155+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-16T14:22:29.170+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_data> on 2023-12-16 14:22:27.349459+00:00
[2023-12-16T14:22:29.177+0000] {standard_task_runner.py:57} INFO - Started process 264 to run task
[2023-12-16T14:22:29.181+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_data_pipeline', 'extract_data', 'manual__2023-12-16T14:22:27.349459+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/taxi_etl.py', '--cfg-path', '/tmp/tmp7x2tg5dt']
[2023-12-16T14:22:29.192+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask extract_data
[2023-12-16T14:22:29.304+0000] {task_command.py:416} INFO - Running <TaskInstance: NYC_Taxi_data_pipeline.extract_data manual__2023-12-16T14:22:27.349459+00:00 [running]> on host 6abb98cbcbb2
[2023-12-16T14:22:29.409+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='islam' AIRFLOW_CTX_DAG_ID='NYC_Taxi_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-16T14:22:27.349459+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-16T14:22:27.349459+00:00'
[2023-12-16T14:22:29.414+0000] {logging_mixin.py:154} INFO - Cleaned dataset already exists on the file system
[2023-12-16T14:22:29.415+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-16T14:22:29.423+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=NYC_Taxi_data_pipeline, task_id=extract_data, execution_date=20231216T142227, start_date=20231216T142229, end_date=20231216T142229
[2023-12-16T14:22:29.440+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-16T14:22:29.464+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
